luigi/parameter.py

```diff
@@ -44,6 +45,25 @@
 _no_value = object()
 
 
+def join_with_commas(values):
+    """ Join all values with a comma.
+
+    This is necessary because ','.join is not picklable but top-level functions
+    are.
+
+    """
+    return ','.join(values)
+
+
+class BatchAggregation(enum.Enum):
+    COMMA_LIST = join_with_commas  # join all values with commas
+    MIN_VALUE = min  # choose just the minimum value by string representation
+    MAX_VALUE = max  # choose just the maximum value by string representation
```

Comment:
I'm not sure how to make these visible on readthedocs. Maybe I should move this documentation to the batch_method parameter description? It seems a little unfortunate to have it there :(